src/main/java/codechicken/nei/ClientHandler.java

```diff
@@ -165,16 +154,19 @@ public static void loadSerialHandlers() {
         if (!file.exists()) {
             try (FileWriter writer = new FileWriter(file)) {
                 NEIClientConfig.logger.info("Creating default serial handlers list {}", file);
-                Collection<String> toSave = Loader.isModLoaded("dreamcraft") ? Collections.singletonList("")
```

Comment:
My intention here is to replace this GTNH check with shipping a dedicated `serialhandlers.cfg` in the config repo.